packages/@lwc/compiler/src/options.ts

```diff
@@ -31,7 +32,9 @@ const DEFAULT_OPTIONS = {
     experimentalComplexExpressions: false,
     disableSyntheticShadowSupport: false,
     enableLightningWebSecurityTransforms: false,
-};
+    targetSSR: false,
+    ssrMode: 'sync',
```

Comment:
Why is the default option `sync` here but `asyncYield` in `@lwc/ssr-compiler/src/index.ts`?

Reply:
Very good question. I wasn't thinking too much about consistency, I guess, just correctness. Happy to change the default mode to be the same in both places.